unit/include/DiscreteFissionPDFTest.h

```diff
@@ -16,21 +16,31 @@
 #define DISCRETEFISSIONPDFTEST_H
 
 //CPPUnit includes
-#include "cppunit/extensions/HelperMacros.h"
 #include "DiscreteFissionPKAPDF.h"
 #include "MooseRandom.h"
 
-class DiscreteFissionPDFTest : public CppUnit::TestFixture
-{
-  CPPUNIT_TEST_SUITE( DiscreteFissionPDFTest );
+#include <gtest/gtest.h>
 
-  CPPUNIT_TEST( sampleFissionPKA );
+class DiscreteFissionPDFTest : public ::testing::Test
```

Comment:
It looks like this class has no state - so there is no point on having it as a test fixture.  You could just take the member functions and make them plain functions in the .C file.